### PR TITLE
Update lira config template

### PR DESCRIPTION
**Pull request by samanehsan**
Changes: 
- Always include the google project (not just when using caas)
- Add new field called "google_pubsub_topic"

### CODE REVIEW

File: config_files/environment.dev [hunk 1]
```diff
@@ -12,6 +12,7 @@ export TENX_SUBSCRIPTION_ID="placeholder_10x_subscription_id"
 export OPTIMUS_SUBSCRIPTION_ID="placeholder_optimus_subscription_id"
 
 export GCLOUD_PROJECT="broad-dsde-mint-dev"
+export GOOGLE_PUBSUB_TOPIC="hca-notifications-${ENVIRONMENT}"
```
Comment (rexwangcc) [inline config_files/environment.dev:15]: question: do we have to create those topics manually in advance? If so, should we create them before merging this PR?

Comment (samanehsan) [reply config_files/environment.dev:15]: Yeah, right now I just created one for dev -- I can do the other environments and add them to this PR too!

Comment (samanehsan) [reply config_files/environment.dev:15]: Done!

**Review by rexwangcc — APPROVED** [review]
👍

**Pull request merged**